### ISSUE TITLE
Where is the AutoUpdater-API?

### ISSUE THREAD

**Issue by thebigsmileXD** [PocketMine/PocketMine-MP#3852]
It would be nice if you would make the www.pocketmine.net/api public, so third party builds can use this and don't get a spammed console "theres a never version of pocketmine"

**Comment by KnownUnown** [comment]
http://pocketmine.net/api/?channel=stable

**Comment by thebigsmileXD** [comment]
.. I mean the real files, with that the PocketMine AutoUpdater class gets if theres a new version or not

**Comment by PEMapModder** [comment]
Do you mean to open the source of the API within www.pocketmine.net?

**Comment by thebigsmileXD** [comment]
Correct.

**Comment by PEMapModder** [comment]
This issue is invalid. It is not related to PocketMine itself. The issue tracker is for PocketMine-MP, not pocketmine.net

**Comment by KnownUnown** [comment]
Even so, I don't see why you can't build it yourself. The JSON format PM is expecting is here:
https://github.com/PocketMine/PocketMine-MP/blob/master/src/pocketmine/updater/AutoUpdater.php#L66
You can use the GitHub public API to fetch releases (and identify stability via tags):
https://developer.github.com/v3/repos/releases/
tl;dr it's not difficult to make your own autoupdater API

**Comment by thebigsmileXD** [comment]
Well, we would have jenkins releases
